pkg/frontend/mysql_cmd_executor.go

```diff
@@ -2278,6 +2266,23 @@ func (mce *MysqlCmdExecutor) doComQuery(sql string) (retErr error) {
 			if txnErr != nil {
 				return txnErr
 			}
+			switch stmt.(type) {
```

Comment:
For this list -- are they exhaustive?   If which are missing?   If it does not fall in them, what happens?

What is the criteria for this list?

Reply:
this list is same as the list in line 2217, and I delete the  *tree.SetVar, *tree.Load case because they are not same as the   other case. And I have checked the list so it is exhaustive. If it does not in this case list, it will send the OKResponse in their own process flow, not in the case branch. And the list is according to the case list in line 2217.

Reply:
I also delete the *tree.BeginTransaction, *tree.CommitTransaction, *tree.RollbackTransaction because it will not run in the code flow because of txn begin,commit,rollback do not need to be committed.

Reply:
You need to comment on this, and have YuanNing review this.

Reply:
ok

Reply:
ok